### DISCUSSION TITLE
Newcomer

### DISCUSSION THREAD

**Discussion by Forme7522** [community/community#50188] [Other Feature Feedback, Questions, & Ideas]
I am still trying to figure all of this out however my phone has been cloned and I'm not sure how to go about it

**Comment by queenofcorgis** [comment]
Hi @Forme7522,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on your device It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
